crm_operating_unit/views/crm_lead_view.xml

```diff
@@ -9,7 +9,10 @@
         <field name="model">crm.lead</field>
         <field name="inherit_id" ref="crm.crm_lead_view_form" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='team_id']" position="after">
```

Comment:
@traitrantrobz Please use `user_id` instead, this field should be visibie in both of lead and opportunity (as v15 did)